Mathlib/Order/RelSeries.lean

```diff
@@ -129,71 +98,61 @@ lemma toList_ne_empty (x : RelSeries r) : x.toList ≠ [] := fun m =>
 
 -- TODO : build a similar bijection between `RelSeries α` and `Quiver.Path`
 
 end RelSeries
 
 namespace Rel
 
-/-- a relation `r` is said to be finite dimensional iff there is a relation series of `r` with the
+/-- A relation `r` is said to be finite dimensional iff there is a relation series of `r` with the
   maximum length. -/
 class FiniteDimensional : Prop where
-  /-- a relation `r` is said to be finite dimensional iff there is a relation series of `r` with the
+  /-- A relation `r` is said to be finite dimensional iff there is a relation series of `r` with the
     maximum length. -/
   exists_longest_relSeries : ∃ (x : RelSeries r), ∀ (y : RelSeries r), y.length ≤ x.length
 
-/-- a relation `r` is said to be infinite dimensional iff there exists relation series of arbitrary
+/-- A relation `r` is said to be infinite dimensional iff there exists relation series of arbitrary
   length. -/
 class InfiniteDimensional : Prop where
-  /-- a relation `r` is said to be infinite dimensional iff there exists relation series of
+  /-- A relation `r` is said to be infinite dimensional iff there exists relation series of
     arbitrary length. -/
   exists_relSeries_with_length : ∀ (n : ℕ), ∃ (x : RelSeries r), x.length = n
 
 end Rel
 
 namespace RelSeries
 
-/-- the longest relational series when a relation is finite dimensional -/
+/-- The longest relational series when a relation is finite dimensional -/
 protected noncomputable def longestOf [r.FiniteDimensional] : RelSeries r :=
-  (inferInstance : r.FiniteDimensional).exists_longest_relSeries.choose
+  Rel.FiniteDimensional.exists_longest_relSeries.choose
 
 lemma length_le_length_longestOf [r.FiniteDimensional] (x : RelSeries r) :
     x.length ≤ (RelSeries.longestOf r).length :=
-  (inferInstance : r.FiniteDimensional).exists_longest_relSeries.choose_spec _
+  Rel.FiniteDimensional.exists_longest_relSeries.choose_spec _
 
-/-- a relation series with length at least `n` if the relation is infinite dimensional -/
+/-- A relation series with length `n` if the relation is infinite dimensional -/
 protected noncomputable def withLength [r.InfiniteDimensional] (n : ℕ) : RelSeries r :=
-  ((inferInstance : r.InfiniteDimensional).exists_relSeries_with_length n).choose
+  (Rel.InfiniteDimensional.exists_relSeries_with_length n).choose
 
 @[simp] lemma length_withLength [r.InfiniteDimensional] (n : ℕ) :
     (RelSeries.withLength r n).length = n :=
-  ((inferInstance : r.InfiniteDimensional).exists_relSeries_with_length n).choose_spec
+  (Rel.InfiniteDimensional.exists_relSeries_with_length n).choose_spec
 
-/-- if a relation on `α` is infinite dimensional, then `α` is inhabited -/
-noncomputable def inhabited_of_infiniteDimensional [r.InfiniteDimensional] : Inhabited α :=
-  ⟨(RelSeries.withLength r 0) 0⟩
+/-- If a relation on `α` is infinite dimensional, then `α` is nonempty. -/
+lemma nonempty_of_infiniteDimensional [r.InfiniteDimensional] : Nonempty α :=
```

Comment:
I think it makes little sense to make a def to produce an arbitrary default element, so I changed it to Nonempty.